components/shader/shadervisitor.cpp

```diff
@@ -267,8 +266,6 @@ namespace Shader
                                 mRequirements.back().mShaderRequired = true;
                             }
                         }
-                        else if (!mTranslucentFramebuffer)
-                            Log(Debug::Error) << "ShaderVisitor encountered unknown texture " << texture;
```

Comment:
This is here for the FFP fallback. There wasn't any good way to make it work without an extra TexEnv stage, but that only works if there's a dummy texture, even when the TexEnv stage doesn't use it. That dummy texture confuses the shader visitor. I see the `recreateShaders` call in the character preview is gone, but I don't remember off the top of my head whether it's possible to end up with them being recreated any other way while that dummy texture's there. It's probably worth double-checking.